third_party/gpus/rocm/BUILD.tpl

```diff
@@ -396,22 +399,24 @@ cc_library(
 
 cc_library(
     name = "rocprofiler-sdk",
-    srcs = glob(["%{rocm_root}/lib/librocprofiler-sdk*.so*"]),
+    data = glob(["%{rocm_root}/lib/librocprofiler-sdk*.so*"]),
```

Comment:
is it ok to link `rocprof-sdk.so` in the runtime? is it supposed to initialized with xla-pjrt together?  cc @cj401-amd

Reply:
unless this change is no problem on jax.profiling and rocprof-sdk, please don't merge it.

Reply:
@alekstheod Is it okay to leave the librocprofiler-sdk in `srcs` for now? Unfortunately, moving it out of `srcs` and adding the linker options didn't fix the busted profiling.